app/experimenter/experiments/models.py

```diff
@@ -477,20 +477,34 @@ def completed_testing(self):
         return self.qa_status
 
     @property
-    def _required_reviews(self):
-        return (
-            self.review_science,
-            self.review_engineering,
-            self.review_qa_requested,
-            self.review_intent_to_ship,
-            self.review_bugzilla,
-            self.review_qa,
-            self.review_relman,
-        )
+    def _conditional_required_reviews_mapping(self):
+        return {"review_vp": self.risk_partner_related}
+
+    @property
+    def _default_required_reviews(self):
+        return [
+            "review_science",
+            "review_engineering",
+            "review_qa_requested",
+            "review_intent_to_ship",
+            "review_bugzilla",
+            "review_qa",
+            "review_relman",
+        ]
+
+    def get_all_required_reviews(self):
+        required_reviews = self._default_required_reviews
+        for review, risk in self._conditional_required_reviews_mapping.items():
+            if risk:
+                required_reviews.append(review)
+        return required_reviews
 
     @property
     def completed_required_reviews(self):
-        return all(self._required_reviews)
+        required_review = self.get_all_required_reviews()
+        return all(
+            list(map(lambda review: getattr(self, review), required_review))
```

Comment:
In general I'd avoid using map/lambda and just stick to list comprehensions:

    return all([getattr(self, r) for r in self.get_all_required_reviews()])